app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/AgendaDetailsScreen.kt

```diff
@@ -390,6 +405,55 @@ private fun AgendaDetailsScreen(
     }
 }
 
+@Composable
+fun AttendeeList(
+    modifier: Modifier = Modifier,
+    attendeeList: List<AttendeeUi>,
+    selectedAttendanceFilter: FilterType
+) {
+    val goingList = attendeeList.filter { it.isGoing }
+    val notGoingList = attendeeList.filter { !it.isGoing }
```

Comment:
These will recalculate on each recomposition (which can happen multiple times a second!). Wrap it in a `remember` block to persist the state across recompositions.

```
val goingList by remember(attendeList) { 
  mutableStateOf(
    attendeeList.filter { it.isGoing }
  )
}
```